api/server/server.go

```diff
@@ -1384,15 +1376,21 @@ func postContainerExecStart(eng *engine.Engine, version version.Version, w http.
 		return nil
 	}
 	var (
-		name             = vars["name"]
-		job              = eng.Job("execStart", name)
-		errOut io.Writer = os.Stderr
+		execName           = vars["name"]
+		errout   io.Writer = os.Stderr
+		stdin    io.ReadCloser
+		stdout   io.Writer
+		stderr   io.Writer
 	)
 
-	if err := job.DecodeEnv(r.Body); err != nil {
+	execConfig := runconfig.ExecConfig{}
```

Comment:
This might be a nit, but I'd prefer to not decode to `ExecConfig` here, instead add something in `api/types` for this, especially since we're really just getting `Detach` and `Tty` here.

@duglin @LK4D4